src/main/java/org/terasology/potions/effect/NoVisibilityEffect.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.potions.effect;
+
+import org.terasology.entitySystem.entity.EntityManager;
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.logic.actions.*;
+import org.terasology.logic.health.*;
+import org.terasology.potions.HerbEffect;
+import org.terasology.potions.events.DrinkPotionEvent;
+
+/**
+ * Once the API for Custom Actions is implemented, the code for NoVisibility Effect goes here.
+ The effect will make the screen blank for the given amount of time.
+ */
+public class NoVisibilityEffect implements HerbEffect {

```

Comment:
If you were compiling the code to make sure all was well you should've seen an error here. Like other effects even though this one is a placeholder pending API improvements anything implementing `HerbEffect` also must implement the `applyEffect` potion. Check out other effects to see how it should look - yours can probably be empty but needs to exist. It looks like you've got the Javadoc for it below, so maybe you just deleted the method header itself?